charts/bulk-scan-orchestrator/tests/test-service.yaml

```diff
@@ -0,0 +1,20 @@
+# ---
```

Comment:
needs an update

Reply:
the problem is that there is nothing that really would indicate a proper deployment unless the health check fails if we can connect to the service bus.
I think that is outside of this card tbh.